src/main/webapp/index.html

```diff
@@ -85,6 +85,10 @@ <h1>Census Viz</h1>
     <div id="result">
       <div id="map-title"></div>
       <div id="more-info"></div>
+      <div id="colors">
+        <input type="color" id="setColor" name="setColor" value="#0000ff" onchange="changeColor(this)">
+        <label for="head">Change the color of the visualization</label>
```

Comment:
I think typically, the label should go before the input, and the label's for attribute should be equal to the id of the input element (so in this case, setColor).

Reply:
fixed!